frontend/src/pages/login/LoginPage.tsx

```diff
@@ -1,14 +1,212 @@
-import { Box, Heading } from '@chakra-ui/react'
+import { FC, useMemo, useState } from 'react'
+import { Link as ReactLink } from 'react-router-dom'
+import {
+  Box,
+  chakra,
+  Flex,
+  Grid,
+  GridItem,
+  Text,
+  useBreakpointValue,
+  Wrap,
+} from '@chakra-ui/react'
 
+import { ReactComponent as BrandLogoSvg } from '~assets/svgs/brand/brand-hort-colour.svg'
+import { ReactComponent as LoginImageSvg } from '~assets/svgs/img-buildscratch.svg'
 import { useAuth } from '~contexts/AuthContext'
-import Button from '~components/Button'
+import { FORM_GUIDE, REPORT_VULNERABILITY } from '~constants/externalLinks'
+import { LANDING_ROUTE } from '~constants/routes'
+import Footer from '~components/Footer'
+import Link from '~components/Link'
+
+import { LoginForm, LoginFormInputs } from './components/LoginForm'
+import { OtpForm, OtpFormInputs } from './components/OtpForm'
+
+export type LoginOtpData = {
+  email: string
+}
+
+const BrandLogo = chakra(BrandLogoSvg)
+const LoginImage = chakra(LoginImageSvg)
+
+export const LoginPage: FC = () => {
```

Comment:
what happened to our `JSX.Element` convention haha

Reply:
BETRAYED BY MY OLD SNIPPET